src/System.Security.Cryptography.X509Certificates/tests/ChainTests.cs

```diff
@@ -632,6 +632,16 @@ public static void Create()
                 Assert.NotNull(chain);
         }
 
+        [Fact]
+        public static void BuildChainInvalidValues()
+        {
+            using (var chain = X509Chain.Create())
+            {
+                Assert.Throws<ArgumentException>(() => chain.Build(null));
+                Assert.Throws<ArgumentException>(() => chain.Build(new X509Certificate2()));
```

Comment:
Use `AssertExtensions.Throws("certificate", () => chain.Build(new X509Certificate2()));` to test that the argument name is as expected across the frameworks / OSes.